hs_access_control/models/utils.py

```diff
@@ -0,0 +1,11 @@
+#############################################
+# utility classes
+#
+# These are used by all access control classes.
+#############################################
+
+
+class PolymorphismError(Exception):
```

Comment:
I can live with an Exception being in utils, but maybe consider a file called exceptions?

Reply:
I've done this in the next PR. The whole goal of this is to allow that one to be reviewed.